backend/src/test/java/reviewme/highlight/domain/HighlightLineTest.java

```diff
@@ -0,0 +1,35 @@
+package reviewme.highlight.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatCode;
+
+import java.util.Set;
+import org.junit.jupiter.api.Test;
+import reviewme.highlight.domain.exception.HighlightIndexExceedLineLengthException;
+
+class HighlightLineTest {
+
+    @Test
+    void 하이라이트_대상_라인의_글자수보다_큰_시작_종료_인덱스_범위를_추가하려고_하면_예외를_발생한다() {
+        // given
+        HighlightLine highlightLine = new HighlightLine(1, "12345");
+
+        // when && then
+        assertThatCode(() -> highlightLine.addRange(4, 5))
+                .isInstanceOf(HighlightIndexExceedLineLengthException.class);
+    }
```

Comment:
```suggestion
    @Test
    void 하이라이트_대상_라인의_글자수보다_큰_시작_종료_인덱스_범위를_추가하려고_하면_예외를_발생한다() {
        // given
        String content = "12345";
        HighlightLine highlightLine = new HighlightLine(1, content);

        // when && then
        assertThatCode(() -> highlightLine.addRange(content.length(), content.length() + 1))
                .isInstanceOf(HighlightIndexExceedLineLengthException.class);
    }
```

[왕사소]
지금의 `assertThatCode(() -> highlightLine.addRange(4, 5))`에 사용되는 4와 5가 의미하는바가 무엇인지를 더 잘 드러낼 수 있을 것 같아요!

Reply:
👍👍반완!